GliaWidgets/Sources/Extensions/Result+Extension.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+
+extension Result {
```

Comment:
We this is needed? `let value = try? result.get()` allows to retrieve underlying value.

Reply:
Just thought of some sugar, but removed.

https://github.com/salemove/ios-sdk-widgets/pull/1112/commits/1cacfa00eac3c54059699956a407c1b4b9823ffd